app/utils.py

```diff
@@ -8,6 +9,23 @@
 from . import config
 
 
+class Hasher:
```

Comment:
It would be nice for this class to at least carry a docstring explaining why it exists and what it's for, e.g.

```python
    """
    Wrapper around hashlib to provide slightly more ergonomic API for hashing
    a combination of data types as might be found in database tables and the like.
    """
```

Reply:
I have added one. Does it look OK, or should I expand on it?

Reply:
I'd like to see a little more if you think it's worthwhile, otherwise this'll do.